pandas/tests/io/test_pytables.py

```diff
@@ -5247,6 +5247,16 @@ def test_query_compare_column_type(self):
                         expected = df.loc[[], :]
                     tm.assert_frame_equal(expected, result)
 
+    def test_read_hdf_series_fixed(self):
+        # GH 16583
+        # Tests that reading a Series saved to an HDF file in 'fixed' format
+        # still works if a mode='r' argument is supplied
+        series = tm.makeFloatSeries()
```

Comment:
can you do this also for a ``format='table'`` (and rename the test slightly). It might duplicate an existing test, but that's ok.

ping when pushed / green. lgtm merge otherwise.